query/parser.go

```diff
@@ -340,8 +384,12 @@ func (p *Parser) insertPropertyKeyValue() {
 	case "from":
 		fallthrough
 	case "to":
-		unix, err := parseDate(value)
-		if err != nil {
+		var unix int64
+		var err error
+		if unix, err = parseDate(value); err == nil {
+			// Valid, so do nothing.
+		} else if unix, err = parseRelativeTime(value, time.Now().UTC()); err != nil {
```

Comment:
Actually, I see this being somewhat problematic, (even though it mostly won't)

1. `now` will be slightly different between `from` and `to`, because we're taking unix timestamps. these values should be minimal but it's still a bit annoying.
2. if there's a delay between parsing & execution, then "now" will stay the same. in fact, it is a valid question whether if we would want be able to execute this query over and over again with the same or different timestamp for `now`.